HopsanCore/CMakeLists.txt

```diff
@@ -61,3 +62,4 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/src/ DESTINATION ${HOPSANCORE_INST
 install_libnumhop_src(${HOPSANCORE_INSTALL_DST}dependencies/libnumhop)
 install_indexingcsvparser_src(${HOPSANCORE_INSTALL_DST}dependencies/indexingcsvparser)
 install_rapidxml_src(${HOPSANCORE_INSTALL_DST}dependencies/rapidxml )
+install_sundials_src({HOPSANCORE_INSTALL_DST}dependencies/sundials)
```

Comment:
Seems to be name difference, 
install_sundials_src vs. install_libsundials_src in sundials.cmake

Reply:
Ops, it is fixed now.